.gitignore

```diff
@@ -6,7 +6,7 @@
 npm
 npm-debug.log
 node_modules/
-
+data3.csv
```

Comment:
Consider renaming your data filenames to reflect the kind of data in the file. 
Instead of ```data3.csv```, you could name the file ```listingData.csv```